.circleci/config.yml

```diff
@@ -420,49 +435,95 @@ jobs:
           keys:
             - '{{ .Environment.CIRCLE_CACHE_VERSION }}-bundle-2.4-{{ checksum ".circleci/bundle_checksum" }}'
       - *step_run_all_tests
+  deploy-release:
+    <<: *job_defaults
+    docker:
+      - *container-2_4
+    steps:
+      - checkout
+      - run:
+          command: |
+            apt-get -y -qq update
+            apt-get -y -qq install awscli
+      - restore_cache:
```

Comment:
We should consider removing this caching step entirely, since this will only run once on tagged builds. This means the cache will basically be useless anyways.